### DISCUSSION TITLE
Laragon localhost wordpress - site images/css not visible on internal network

### DISCUSSION THREAD

**Discussion by agg241224711** [leokhoa/laragon#891] [Q&A]
Noob Q please: when browsing wordpress on local IP within same WiFi - wordpress site text appears - but none of the CSS and images.
They all have the url written as "localhost" instead of the local IP of the PC serving LARAGON Apache from (e.g.) 192.168.1.207
Any JEDIs can help pls?

**Comment by Sophist-UK** [comment]
According to [StackOverflow](https://stackoverflow.com/questions/29697144/change-site-url-and-wordpress-url-in-localhost)...

>  go to the wp-config.php File and place this code:
   ```
define('WP_HOME','http://192.168.1.207');
define('WP_SITEURL','http://192.168.1.207');
   ```
